TakeArguments.java

```diff
@@ -0,0 +1,10 @@
+class TakeArguments{
+public static void main(String[] args){
+if (args.length==1){
```

Comment:
If there is a single argument, then everything works as expected (args.length is 1) but what happens if there are multiple arguments (args.length = 3)? Will the output be the argument, or "No Arguments are given"?

Reply:
if args.length=3 then the output will be "No Arguments given!" .